sycl/include/sycl/ext/intel/esimd/math.hpp

```diff
@@ -101,10 +101,10 @@ __esimd_abs_common_internal(simd<TArg, SZ> src0) {
 }
 
 template <typename TRes, typename TArg>
-ESIMD_NODEBUG ESIMD_INLINE
-    std::enable_if_t<detail::is_esimd_scalar<TRes>::value &&
-                         detail::is_esimd_scalar<TArg>::value,
-                     TRes>
+ESIMD_NODEBUG
+    ESIMD_INLINE std::enable_if_t<detail::is_esimd_scalar<TRes>::value &&
```

Comment:
nit: there are some whitespace changes of this pattern that in my opinion decrease readability, i would expect we only need to touch the rdtsc function and not these